docs/manual/file_mode.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: rpm.org - File Mode
+---
+# File Mode
+
+RPM supports a variety of compression methods through external libraries for
+its internal I/O operations.  For ease of use, the [RPMIO
+API](https://ftp.osuosl.org/pub/rpm/api/4.17.0/group__rpmio.html) offers
```

Comment:
This points out two things (neither of which is the fault of this PR):
- I forgot to upload 4.18 API docs
- the API docs need to go into the docs directory too so we link between API and other docs without this kind of thing (that is now outdated from the go, and will require cumbersome constant updating).

Reply:
Indeed, I was actually expecting somebody to notice this hardcoded 4.17.0 link here :smile: I was thinking of having something like the "latest" symlink there, for cases like this where you need to link the API docs. Or, as you said, moving it to docs.